docs/spark/tutorials/get-started.md

```diff
@@ -143,48 +155,34 @@ Open *Program.cs* in Visual Studio Code, or any text editor, and replace all of
 ```csharp
 using Microsoft.Spark.Sql;
 
-namespace MySparkApp
+namespace HelloSpark
```

Comment:
Is there a reason we're changing the coding example here? Should we have this coding example align with https://dotnet.microsoft.com/learn/data/spark-tutorial/code? If so, perhaps we can either leave this code example as it was, or consider changing what's on the .NET website Spark landing page?

Reply:
I think we have the `HelloSpark` example in [SparkDotnet github repo](https://github.com/dotnet/spark/blob/master/docs/getting-started/windows-instructions.md#authoring-a-net-for-apache-spark-app). But I agree we should keep this coding example consistent with [SparkDotnet Tutorial](https://dotnet.microsoft.com/learn/data/spark-tutorial/code). I would suggest to use `work_count_example` since this is classic intro example for Big Data World.